client/src/app/pages/applications/application-form/useApplicationForm.ts

```
@@ -0,0 +1,353 @@
+        })
+        .when("artifact", {
+          is: (artifact: string) => artifact?.length > 0,
+          then: (schema) => schema.required("This field is required."),
+          otherwise: (schema) => schema.trim(),
+        }),
+
+      // asset repository fields
+      assetKind: string().oneOf(["", "git", "subversion"]),
+      assetRepository: string().when("assetKind", {
+        is: (kind: string) => kind !== "",
+        then: (schema) => schema.repositoryUrl("kind").required(),
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)